docs/operating-scylla/nodetool-commands/restore.rst

```diff
@@ -43,8 +44,15 @@ Options
 * ``--keyspace`` - Name of the keyspace to load SSTables into
 * ``--table`` - Name of the table to load SSTables into
 * ``--nowait`` - Don't wait on the restore process
+* ``--scope`` - Use specified load-and-stream scope
 * ``<sstables>`` - Remainder of keys of the TOC (Table of Contents) components of SSTables to restore, relative to the specified prefix
 
+The `scope` parameter describes the list of nodes from the cluster to which restoring will
```

Comment:
nits:
- add **optional** `scope` parameter
- s/list/subset/ (since we don't have an option to pass an explicit list of nodes)